statics/js/components/capture-form.js

```diff
@@ -162,12 +184,51 @@ Vue.component('capture-form', {
           self.resetQueryNodes();
           self.queryNodes = nodes;
           self.highlightQueryNodes(true);
+          if (nodes.length == 1) {
+            self.typeAllowed = true;
+            self.nodeType = nodes[0].Metadata["Type"];
+          } else if (nodes.length > 1) {
+            self.typeAllowed = false;
+          } else {
+            self.typeAllowed = true;
```

Comment:
I think we should only set it to true if nodes.length == 1, otherwise if the nodes selected are not on the same layer2 we will get no node and will be able to select something while the capture is not matching anything

Reply:
done